WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
Customisation is not something I'm opposed to, but it isn't a priority. Do remember that the more options we add, the more weights and setting we have to maintain.